stackslib/src/net/api/poststackerdbchunk.rs

```diff
@@ -54,6 +54,15 @@ use crate::net::{
 };
 use crate::util_lib::db::{DBConn, Error as DBError};
 
+/// Event structure for newly-arrived StackerDB data
```

Comment:
Can you instead put this into `stackslib/src/chainstate/stacks/events.rs`?

Reply:
Done :)